test/extended/gssapi.sh

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env bash
+#
+# Extended tests for logging in using GSSAPI
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STARTTIME="$(date +%s)"
+
+PROJECT='gssapiproxy'
```

Comment:
I feel like vars that are passed to the containers and used as globals (i.e. called from functions that don't even know they exist) throughout the script should be in caps.